plugin/discovery/get_test.go

```diff
@@ -216,7 +228,6 @@ func TestCheckProtocolVersions(t *testing.T) {
 			if err == nil {
 				t.Fatal("succeeded; want error")
 			}
```

Comment:
When doing a table-based test like this we usually try to write it with `t.Run` in the loop so that it can run all of the test cases even if one of them fails. I think that could work here too, though I guess there isn't an obvious thing to use as the subtest name here. :thinking:

Reply:
thank you for the review! ah yeah i'm a fan of `t.Run` but was hoping to make minimal changes since i was already making an assumption on the intent of the test. It's likely with what i'm working that i'll revisit this and may include the changes then!